actiontext/lib/generators/action_text/install/install_generator.rb

```diff
@@ -49,7 +49,7 @@ def create_actiontext_files
       end
 
       def create_migrations
-        rails_command "railties:install:migrations FROM=active_storage,action_text"
+        rails_command "railties:install:migrations FROM=active_storage,action_text", inline: true
```

Comment:
This line was not hit by the tests I ran locally, so I am currently unsure of its impact.